install/kubernetes/cilium/charts/agent/values.yaml

```diff
@@ -8,3 +8,14 @@ maxUnavailable: 2
 monitor:
   enabled: false
   eventTypes: []
+  # Specifies the resources for the monitor sidecar container if activated
+  resources: {}
+
+# Specifies the resources for the agent container
+resources: {}
+
+# Specifies the resources for the clean-cilium-state init container
+initResources:
```

Comment:
I am not particularly fond of this naming but also could not come up with anything better. If you have a suggestion I would happily change it.